### ISSUE TITLE
No subreddit pages?

### ISSUE THREAD

**Issue by Samisafool** [gurnec/removeddit#4]
unddit.com/r/linux doesn't work for example

**Comment by gurnec** [comment]
That's correct, it's unsupported. This is [mentioned in the FAQ](https://www.unddit.com/) (second Q).

One reason is that I'm cheap. Right now, the site is very inexpensive to operate. Supporting subreddit pages would require a SQL backend which would cost more. Another is that I'm lazy—I'd have to figure out if the Removeddit API code which used to support this would even work today.

For now one should use Reveddit.

For anyone interested in seeing subreddit support, please downvote this comment (seriously). If it gets enough downvotes, maybe I'll revisit this. In the mean time, I'll leave the issue open but marked `wontfix`.

**Comment by 07416** [comment]
[reveddit doesn't restore content deleted by the user](https://www.reveddit.com/about/faq/#user-deleted), which is a large limitation since some post with the intent of removing later, for instance after:

* their question has been "answered"
* wipe their post history frequently with a script
* the post wasn't overall positively received
* spammers attempting to evade moderators

Albeit likely less common now with the official mobile app being the largest platform (meaning no real access to scripts or macros), I still frequently see removed parent-level comments (which can have dozens of direct replies ,hundreds or even thousands in total) in more or less every active post with more than two days of age.
